### PR TITLE
Ensure we use the discovered image in computes if not other one given.

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date

### CODE REVIEW

**Review by cjeanner — COMMENTED** [review]
/approve

**Comment by cjeanner** [comment]
recheck

**Review by raukadah — APPROVED** [review]

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

File: ci_framework/roles/libvirt_manager/tasks/deploy_edpm_compute.yml [hunk 1]
```diff
@@ -45,7 +45,8 @@
           'EDPM_COMPUTE_RAM': compute_config.memory,
           'EDPM_COMPUTE_DISK_SIZE': compute_config.disksize,
           'CRC_POOL': compute_config.image_local_dir,
-          'DISK_FILEPATH': cifmw_libvirt_manager_basedir + "/workload/edpm-compute-" + item + ".qcow2"
+          'DISK_FILEPATH': cifmw_libvirt_manager_basedir + "/workload/edpm-compute-" + item + ".qcow2",
+          'CENTOS_9_STREAM_URL': compute_config.image_url | default(cifmw_discovered_image_url)
```
Comment (dasm) [inline ci_framework/roles/libvirt_manager/tasks/deploy_edpm_compute.yml:49]: Where is this new `CENTOS_9_STREAM_URL` used?

```
[dasm@fedora ci-framework]$ grep -rn CENTOS_9_STREAM_URL *
[dasm@fedora ci-framework]$ 
```

**Comment by cjeanner** [comment]
Seems to have passed in testproject. Let's get it in without obsessing on Zuul:
https://review.rdoproject.org/r/c/testproject/+/49693/2#message-6363003667977eb1ba8ace06c74d3a2fd029faa2

Green result: https://review.rdoproject.org/zuul/build/c7fb61c419d24097a9cbdcaea01ff056

File: ci_framework/roles/libvirt_manager/tasks/deploy_edpm_compute.yml (hunk 1 above)
Comment (pablintino) [reply ci_framework/roles/libvirt_manager/tasks/deploy_edpm_compute.yml:49]: Inside install_yamls :) Is not in out repo

Comment (dasm) [reply ci_framework/roles/libvirt_manager/tasks/deploy_edpm_compute.yml:49]: ack

**Pull request merged**